packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -69,15 +70,24 @@ function createScale(scale: Scale | null, domain: VDomain | undefined): Scale {
 
 // This function both extract the domain and checks that the function has only one parameter.
 function extractDomainFromOneArgFunction(fn: Lambda): VDomain | undefined {
-  const parameters = fn.getParameters();
-  if (parameters.length !== 1) {
+  const counts = fn.paramCounts();
+  if (!includes(counts, 1)) {
     throw new REOther(
-      `Plots only work with functions that have one parameter. This function has ${parameters.length} parameters.`
+      `Plots only work with functions that have one parameter. This function only supports [${counts.join(
+        ","
```

Comment:
This is similar to an error in Calculator that I commented on, but differently formatted.

How about a common `fn.getParameterCountsString()`?